app/components/modals/modal-base.js

```diff
@@ -85,13 +86,13 @@ export default UiModal.extend({
       }
     };
 
-    const options = this.options ? merge(defaultOptions, this.options) : defaultOptions;
+    const options = this.options ? merge(this.defaultOptions, this.options) : this.defaultOptions;
     assign(settings, options);
   },
 
   didInitSemantic() {
     if (this.isOpen) {
-      $(this.element).modal('show');
+      $(this.element).modal({...this.defaultOptions}).modal('show');
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6392563)